sympy/physics/mechanics/body.py

```diff
@@ -0,0 +1,168 @@
+from sympy import Symbol
+from sympy.physics.mechanics import (RigidBody, Particle, ReferenceFrame,
+                                     inertia)
+from sympy.physics.vector import Point, Vector
+
+__all__ = ['Body']
+
+
+class Body(RigidBody, Particle):
+    """
+    A Body which can be connected by joints.
```

Comment:
It still says "joints". Here is a suggestion.

A Body represents either a rigid body or particle in classical mechanics. Bodies have a body-fixed reference frame, a mass, a mass center and possibly a body-fixed inertia.